grakn-kb/src/main/java/ai/grakn/kb/internal/concept/SchemaConceptImpl.java

```diff
@@ -135,24 +137,58 @@ public T sup() {
         return cachedSuperType.get();
     }
 
+
+    /**
+     *
+     * @return The supertypes of this
+     * TODO: add option to return unfiltered supertypes
```

Comment:
Nah no need. Kill the TODO. Also these java docs are already declared on the interface so no need to duplicate them here.

Reply:
Removed